opencti-platform/opencti-front/src/private/components/common/files/workbench/WorkbenchFileContent.jsx

```diff
@@ -366,6 +366,15 @@ const WorkbenchFileContentComponent = ({
     if (file.metaData.entity_id && file.metaData.entity) {
       currentEntityId = file.metaData.entity_id;
     }
+    // update entity container objects_refs
+    if (currentEntityId) {
+      const currentEntityContainer = containers.find((container) => container.x_opencti_id === currentEntityId);
+      if (currentEntityContainer) {
+        const currentEntityObjectRefs = currentEntityContainer.object_refs?.length ? currentEntityContainer.object_refs : [];
```

Comment:
Could be simplified : currentEntityContainer.object_refs = R.uniq([...(currentEntityContainer.object_refs ?? []), ...objectIds ]);

Reply:
`currentEntityContainer.object_refs ?? []` doesn't work because I had cases where currentEntityContainer.object_refs was an empty object {}, not an empty array. So I had to test that the object has a length to be sure that it's an array.

Reply:
That's weird... In that case use Array.isArray(currentEntityContainer). It is better because it tells what we want to test (if it actually is an array)

Reply:
Done